### PR TITLE
disco: temporarily disable disco

### PR DESCRIPTION
**Pull request by oliver-sanders**
* Tornado 6+ does not support WSGI.
* Disable it for now, fix later.
* See https://github.com/metomi/rose/issues/2492

### CODE REVIEW

**Review by MetRonnie — COMMENTED** [review]
Should this have a changelog entry?

**Comment by oliver-sanders** [comment]
Eh, could do, it's only us who care about this feature.

**Comment by oliver-sanders** [comment]
Put one in for the hell of it.

File: CHANGES.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ Rose release 61. This is a __beta pre-release__.
 
 For use with Cylc see [Cylc-Rose](https://github.com/cylc/cylc-rose).
 
+[#2493](https://github.com/metomi/rose/pull/2493) -
+Disable the `rosie disco` web service pending a fix to the tornado version
+conflict with cylc-uiserver, see https://github.com/metomi/rose/pull/2493
+
```
Comment (MetRonnie) [inline CHANGES.md:22]: Shouldn't it be under 2.0b3?

**Review by MetRonnie — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (oliver-sanders) [reply CHANGES.md:22]: Ha, I guess we didn't make any changes for b2!

**Review by datamel — APPROVED** [review]
I've checked out the branch and can confirm this is disabled. I'm happy for merge once @MetRonnie is.

File: CHANGES.md (hunk 1 above)
Comment (oliver-sanders) [reply CHANGES.md:22]: Created a b3 section for this entry.

**Review by MetRonnie — APPROVED** [review]
👍

**Comment by MetRonnie** [comment]
Hmm

https://github.com/metomi/rose/runs/4164101189?check_suite_focus=true#step:5:161
```
ERROR: No matching distribution found for cylc-flow==8.0b3.*
```

**Comment by oliver-sanders** [comment]
Might as well squeeze a fix for that in here too...

**Comment by oliver-sanders** [comment]
Docs passing.

**Pull request merged**